### PR TITLE
add 0.75 release notes

### PR DESCRIPTION
**Pull request by aclevername**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: docs/release_notes/0.75.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Release 0.75.0
+
+
+## Features
+
+- Add --config flag to get labels & nodegroups (#4469) (#4465)
```
Comment (nikimanoledaki) [inline docs/release_notes/0.75.0.md:6]:
```suggestion
- Add --config-file flag to get labels & nodegroups (#4469) (#4465)
```

File: docs/release_notes/0.75.0.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- Add option to skip endpoint creation completely (#4457)
```
Comment (cPu1) [inline docs/release_notes/0.75.0.md:7]: This might not be clear without mentioning private clusters. I think we should rephrase it to:

```suggestion
- Add option to skip endpoint creation for private clusters (#4457)
```

**Review by cPu1 — APPROVED** [review]

**Review by nikimanoledaki — APPROVED** [review]

**Comment by Himangini** [comment]
which one of these is the brew tap ticket?

File: docs/release_notes/0.75.0.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## Improvements
+
+- Improve kube-proxy updating to always get the latest version&build #4344 (#4440)
+- Document logs:PutRetentionPolicy in IAM policies doc (#4456)
+
+## Bug Fixes
+
+- Add option to skip endpoint creation completely (#4457)
+- Fix update logic in iam service account (#4460)
```
Comment (Himangini) [inline docs/release_notes/0.75.0.md:17]:
```suggestion
- Fix update logic in iam service account (#4460)
- Brew Deprecation Warning: Calling bottle :unneeded is deprecated (#4458)
```

File: docs/release_notes/0.75.0.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Release 0.75.0
```
Comment (Himangini) [inline docs/release_notes/0.75.0.md:1]: shouldn't this be Release 0.74.0 ?

Comment (Himangini) [reply docs/release_notes/0.75.0.md:1]: my bad read the releases page wrong https://github.com/weaveworks/eksctl/releases 🙈  
ignore me :D
